lib/common/common/src/mmap_hashmap.rs

```diff
@@ -294,6 +318,15 @@ impl Key for str {
         //    validity of `str`/`String` types.
         buf.get(..self.len()) == Some(AsBytes::as_bytes(self)) && buf.get(self.len()) == Some(&0xFF)
     }
+
+    fn align() -> usize {
```

Comment:
Nit: we can replace this method it with an associated constant.

I've pushed the fix.